app/state_machines/efile_submission_state_machine.rb

```diff
@@ -150,18 +155,19 @@ class EfileSubmissionStateMachine
         recovery_rebate_credit: [benefits.eip1_amount, benefits.eip2_amount].compact.sum,
         third_stimulus_amount: benefits.eip3_amount,
       })
+    elsif submission.is_for_state_filing?
+      StateFile::AfterTransitionMessagingService.new(submission.data_source).send_efile_submission_accepted_message
+      send_mixpanel_event(submission, "fyst_efile_return_accepted")
```

Comment:
@embarnard Minor note, but I see that EfileSubmissionStateMachine. qualifier is missing here compared to line 134. Also the mixpanel events are inconsistent about using fyst vs state_file

Reply:
I think we should keep the mixpanel event names consistent for sure.  Now, fyst and statefile makes sense, but this is adding a bit of complexity for posterity.  We should probably keep using statefile as that is what the codebase does.  Unless we want to make a separate PR to update all statefile references to fyst.